build-tools/scripts/nunit3-console.cmd.in

```diff
@@ -0,0 +1,2 @@
+@echo off
+@NUNIT_CONSOLE_PACKAGE_PATH@\tools\nunit3-console.exe %1% %2% %3% %4% %5% %6% %7% %8% %9%

```

Comment:
I think this should use `%*` instead?

Reply:
maybe, I wasn't sure if it was a thing and I couldn't find the docs :) It's been 25+ years since I wrote a DOS batch file :)